os_migrate/plugins/module_utils/serialization.py

```diff
@@ -16,7 +16,7 @@ def new_resources_file_struct():
     return data
 
 
-def add_or_replace_resource(resources, resource_or_data):
+def add_or_replace_resource_old(resources, resource_or_data):
```

Comment:
We can delete this `_old` function now, right? The refactored function looks good, and the rest of the patch too. :+1: